include/RAJA/policy/sequential/thread.hpp

```diff
@@ -0,0 +1,47 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   RAJA header file defining sequential thread operations.
+ *
+ ******************************************************************************
+ */
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-25, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_policy_sequential_thread_HPP
+#define RAJA_policy_sequential_thread_HPP
+
+#include "RAJA/config.hpp"
+
+#include "RAJA/util/macros.hpp"
+
+#include "RAJA/policy/thread_auto.hpp"
+
+namespace RAJA
+{
+
+RAJA_SUPPRESS_HD_WARN
+template<>
+RAJA_HOST_DEVICE RAJA_INLINE int get_max_threads(seq_atomic)
```

Comment:
Should this use the thread policy? Or does this need to work with thread and atomic policies?